packages/cli-kit/src/public/node/themes/factories.ts

```diff
@@ -52,7 +52,9 @@ export function buildThemeAsset(asset?: RemoteAssetResponse): ThemeAsset | undef
 
   const {key, checksum, attachment, value} = asset
   // Note: for attachments, this is the size of the base64 string, not the real length of the file
-  const stats = {size: (value || attachment).length, mtime: Date.now()}
```

Comment:
Oh, I thought they'd be empty string at this point. You can also just do:

```ts
  const stats = {size: (value || attachment || '').length, mtime: Date.now()}
```

Reply:
In fact, the types suggest these are already strings... I guess we are passing undefined as `any` directly from what we get from the cloud, thus bypassing TS checks?

Reply:
Indeed, possibly making `value` and `attachment` nullable in the `RemoteAssetResponse` would represent a bit better their meaning.

Reply:
`size` is an empty string when the file is empty, so I changed this expression to `value ?? attachment` which returns `''`

`value || attachment` was returning `undefined`